### PR TITLE
Set protobuf and gogo dependencies LS-7541

### PR DESCRIPTION
**Pull request by jmacd**
Set required versions for protobuf and gogo. Protobuf was set to "master".

### CODE REVIEW

File: version.go [hunk 1]
```diff
@@ -0,0 +1,3 @@
+package lightstep
```
Comment (jmacd) [inline version.go]: `dep` doesn't work without this.

File: Gopkg.toml [hunk 2]
```diff
@@ -1,10 +1,10 @@
 [[constraint]]
   name = "github.com/gogo/protobuf"
-  version = "0.5.0"
+  version = "1.0.0"
```
Comment (jmacd) [inline Gopkg.toml]: We could go as high as 1.2.1. Internal is on 1.0.0.

File: Gopkg.toml [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
 
 [[constraint]]
-  branch = "master"
   name = "github.com/golang/protobuf"
+  version = "1.0.0"
```
Comment (jmacd) [inline Gopkg.toml:7]: This has to change. We can go as high as 1.3.0, or we could stop using protobuf entirely.

File: Gopkg.toml (hunk 2 above)
Comment (MatthewDolan) [reply Gopkg.toml]: I am hoping to bump internal to 1.2.1. I can send you that pr.

**Review by MatthewDolan — APPROVED** [review]

**Pull request merged**